### PR TITLE
ISPN-13465 server config in yaml json

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-13465

Needs backport to 13.0.x

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/proc_server_identity_kerberos.adoc [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+* Have Kerberos service account principals.
 
 [NOTE]
 ====
-_keytab_ files can contain both user and service account principals. However,
-{brandname} servers use service account principals only. As a result,
-{brandname} servers can provide identity to clients and allow clients to
-authenticate with Kerberos servers.
+_keytab_ files can contain both user and service account principals.
+However, {brandname} Server uses only service account principals only which means it can provide identity to clients and allow clients to authenticate with Kerberos servers.
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/proc_server_identity_kerberos.adoc:13]: Redundant "only"

File: documentation/src/main/asciidoc/topics/ref_server_realm_ldap.adoc [hunk 2]
```diff
@@ -11,12 +11,27 @@ LDAP servers can have different entry layouts, depending on the type of server a
...
 
-.Endpoint authentication mechanisms
+[IMPORTANT]
+====
+The principal for LDAP connections must have necessary privileges to perform
+LDAP queries and access specific attributes.
+====
 
-When you configure {brandname} Server to use an LDAP realm, you can configure endpoints to use the following authentication mechanisms:
+As an alternative to verifying user credentials with the `direct-verification` attribute, you can specify an LDAP password with the `user-password-mapper` element.
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/ref_server_realm_ldap.adoc:20]: "you can specify the which LDAP attribute to use when validating the password with the `user-password-mapper` element.

**Review by tristantarrant — CHANGES_REQUESTED** [review]
Just two very minor changes.

File: documentation/src/main/asciidoc/topics/proc_server_identity_kerberos.adoc (hunk 1 above)
Comment (oraNod) [reply documentation/src/main/asciidoc/topics/proc_server_identity_kerberos.adoc:13]: nice catch!

**Comment by oraNod** [comment]
@tristantarrant Should I go ahead and squash to make life easy with the backport?

**Comment by tristantarrant** [comment]
Do it

**Comment by oraNod** [comment]
Done. Here's the backport as well: https://github.com/infinispan/infinispan/pull/9830

**Comment by tristantarrant** [comment]
Merged

**Pull request merged**